tools/plot/interactive_objects_evidence_over_time.py

```diff
@@ -0,0 +1,766 @@
+        cam_a = self.plotter.renderers[1].GetActiveCamera()
+        cam_b = self.plotter.renderers[2].GetActiveCamera()
+        self.align_camera(cam_a, cam_b)
+        self.plotter.render()
+
+    def mlh_callback(self, widget: Button, event: str) -> None:
+        """Align the MLH renderer's camera with the groundtruth renderer."""
+        cam_a = self.plotter.renderers[1].GetActiveCamera()
+        cam_b = self.plotter.renderers[2].GetActiveCamera()
+        self.align_camera(cam_b, cam_a)
+        self.plotter.render()
+
+    def slider_callback(self, widget: Slider2D, event: str) -> None:
+        """Respond to slider step change by updating all subplots.
+
+        Note: This function is throttled to prevent recursion depth errors
+            while continue to be responsive.
+        """
+        val = round(widget.GetRepresentation().GetValue())
+        if (
+            val != self.curr_slider_val
+            and time.time() - self.last_call_time > self.throttle_time
+        ):
+            self.gt_sim(plotter=self.plotter, step=val)
+            self.mlh_sim(plotter=self.plotter, step=val)
+            self.evidence_plotter(plotter=self.plotter, step=val)
+
+            self.curr_slider_val = val
+            self.last_call_time = time.time()
+            self.render()
+
+    def render(self, resetcam: bool = False) -> None:
+        """Render the visualization layout.
+
+        Args:
+            resetcam: If True, resets camera for all renderers.
+        """
+        self.plotter.render()
+        self.plotter.at(self.gt_sim.renderer_ix).show(
+            axes=self.gt_sim.axes_dict(),
+            resetcam=resetcam,
+            interactive=False,
+        )
+        self.plotter.at(self.mlh_sim.renderer_ix).show(
+            axes=self.mlh_sim.axes_dict(), resetcam=resetcam, interactive=False
+        )
+        self.plotter.at(self.evidence_plotter.renderer_ix).show(
+            axes=self.evidence_plotter.axes_dict(),
+            camera=self.evidence_plotter.cam_dict(),
+            resetcam=False,
+            interactive=True,
+        )
+
+
+def plot_interactive_objects_evidence_over_time(
+    exp_path: str,
+    data_path: str,
+    learning_module: str,
+) -> int:
+    """Interactive visualization for unsupervised inference experiments.
+
+    This visualization provides a 3-pane renderers to allow for inspecting the objects,
+    MLH, and sensor locations while stepping through the maximum evidence scores for
+    each object.
+
+    Args:
+        exp_path: Path to the experiment directory containing the detailed stats file.
+        data_path: Path to the root directory of YCB object meshes.
+        learning_module: The learning module to use for extracting evidence data.
+
+    Returns:
+        Exit code.
+    """
+    if not Path(exp_path).exists():
+        logger.error(f"Experiment path not found: {exp_path}")
+        return 1
+
+    data_path = Path(data_path).expanduser()
```

Comment:
`data_path` is expected to be a `str` and not `Path`.

```suggestion
    data_path = str(Path(data_path).expanduser())
```